### ISSUE TITLE
help me pls

### ISSUE THREAD

**Issue by axen1** [kingpin1905/LoLAutoHighlight#1]
cv2.error: OpenCV(4.8.0) D:\a\opencv-python\opencv-python\opencv\modules\imgproc\src\templmatch.cpp:588: error: (-215:Assertion failed) corr.rows <= img.rows + templ.rows - 1 && corr.cols <= img.cols + templ.cols - 1 in function 'cv::crossCorr'

**Comment by kingpin1905** [comment]
This error usually happens when the input video is too small. Is the video in 1080p?
Could you include more info if you changed some stuff?

**Comment by axen1** [comment]
720P video. For testing, I downloaded a video from youtube and it downloaded in 720p. I get this error when I run the python script.

**Comment by kingpin1905** [comment]
Yep. This is it! The program tries to scan outside of the 720p video. Try using a 1080p video and tell if it works.

On my side, I'll try to make it work for 4k and 720p

**Comment by axen1** [comment]
Hello result you look or see pls 
![image](https://github.com/kingpin1905/LoLAutoHighlight/assets/31828262/1f480340-2207-45c9-9407-76d8d8ff1ff8)

**Comment by kingpin1905** [comment]
Thank you. I'll look into that.

**Comment by ManuelRoyBo** [comment]
You got this error because the script found no events (kills death, etc).

This is because the video you input isn't a spectator video, it is the video from a player's POV.
My script only work with spectator videos

**Comment by ManuelRoyBo** [comment]
This:
https://youtu.be/gapSIdUT8Us?si=AnoraQRc-PGT0rZw

**Comment by ManuelRoyBo** [comment]
This is the same error as the one you originaly sent. Your video is probably not in 1080p
